src/Traits/LogsActivity.php

```diff
@@ -46,6 +51,16 @@ protected static function bootLogsActivity()
         });
     }
 
+    public function getSubmitEmptyLogs (): bool
+    {
+        return is_null($this->submitEmptyLogs) ? true : $this->submitEmptyLogs;
```

Comment:
`isset()` instead of `is_null()`.
And does it have to be non-static? All other settings are static or have setters.